pkg/breakglasscredential/flags.go

```diff
@@ -113,13 +113,15 @@ func CreateBreakGlass(cluster *cmv1.Cluster,
 func CreateBreakGlassConfig(args *BreakGlassCredentialArgs) (*cmv1.BreakGlassCredential, error) {
```

Comment:
@chenz4027 Could you add unit tests to cover the new functionality in this function. It would be good to have tests for:

- Passing a `nil` args which should result in no error and an empty `BreakGlassCredential`
- Passing args with just `username` set
- Passing args with just `expirationDuration` set

cheers,

Rob

Reply:
Will work on it, thanks @robpblake

Reply:
Updated - @robpblake would you mind taking another look?